Observer/Order.php

```diff
@@ -30,26 +31,28 @@ public function execute(Observer $observer)
         try {
             $order = $observer->getEvent()->getOrder();
             $storeId = $order->getStoreId();
-
+    
             if (!$this->helper->isEnabled($storeId)) {
                 return;
             }
-
-            $this->helper->callBatchApi($storeId, [$order]);
-
-            // If order is made from the FrontEnd
-            if ($order->getRemoteIp()) {
-                $identify = array(
-                    'id' => $order->getCustomerEmail(),
-                    'params' => array(
-                        'email'      => $order->getCustomerEmail(),
-                        'first_name' => $order->getBillingAddress()->getFirstname(),
-                        'last_name'  => $order->getBillingAddress()->getLastname(),
-                        'name'       => $order->getBillingAddress()->getName()
-                    )
-                );
-                $this->helper->addSessionEvent('identify', 'identify', $identify);
+    
+            $token         = $this->helper->getApiToken($storeId);
+            $platform      = 'Magento ' . $this->helper->metaData->getEdition() . ' ' . $this->helper->metaData->getVersion();
+            $pluginVersion = $this->helper->moduleList->getOne($this->helper::MODULE_NAME)['setup_version'];
+    
+            $client        = new Client($token, $platform, $pluginVersion);
+    
+            if (!trim($order->getCustomerEmail())) {
```

Comment:
why?

Reply:
Our current staging has this condition, there was and issue with orders without emails being synced and we implemented this to exclude them.